concepts/comparisons/about.md

```diff
@@ -1,2 +1,197 @@
-#TODO: Add about for this concept.
+# About
 
+A [comparison operator][comparisons] in Python (_also called a Python relational operator_), looks at the values of two operands and returns `True` or `False` based on whether the `comparison` condition is met.
+The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, and `"!="`.
+They all have the same priority (which is higher than that of the Boolean operations).
+
+```python
+>>> 7 > 5
+True
+>>> 99 < 100
+True
+>>> 4 < 4
+False
+>>> 4 <= 4
+True
+>>> 1 >= 1
+True
+>>> 5 == 5
+True
+>>> 6 != 6  # not equal to
+False
+>>> 5 != 6
+True
+>>> not 3 == 3  # interpreted as not(3 == 3)
+False
+```
+
+## Comparison Chaining
+
+Comparisons can be chained arbitrarily, e.g., `x < y <= z` is equivalent to `x < y` `and` `y <= z`, except that `y` is evaluated only once (but in both cases `z` is _not_ evaluated at all when `x < y` is found to be `False`).
+This is also called `short-circuit` evaluation which means the execution is stopped if the truth value of the expression has already been determined.
+Note that the evaluation of expression takes place from left to right.
+In python, short circuiting is supported by various boolean operators, functions and, in this case, comparison chaining.
+
+Also unlike `C`, expressions like `a < b < c` have the interpretation that is conventional in mathematics.
+
+```python
+>>> x = 2
+>>> y = 5
+>>> z = 10
+>>> x < y < z
+True
+>>> x < y > z
+False
+>>> x > y < z
+False
+```
+
+## Comparison between different data types
+
+Since everything in Python is an `object`, things can get interesting when objects of different types are compared.
+For example, the `str` value of a number is considered completely different from the `integer` or `floating-point` value.
+However, an `integer` **can** be considered equal to a `float`, as they are both numeric types that Python can implicitly convert to compare.

```

Comment:
```suggestion
Python makes this distinction because strings are text, while `integers` and `floats` are numeric types.

However, an `integer` **can** be considered equal to a `float`, as they are both numeric types that Python can implicitly convert to compare.

```